be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/PopchaValidator.scala

```diff
@@ -41,20 +38,13 @@ sealed class PopchaValidator(dbActorRef: => AskableActorRef) extends MessageData
           _ <- checkResponseMode(rpcMessage, authenticate.responseMode, validationError(s"Invalid response mode ${authenticate.responseMode}"))
           _ <- checkChannelType(rpcMessage, ObjectType.POPCHA, channel, dbActorRef, validationError(s"Incorrect channel $channel for popcha authentication message"))
           _ <- checkAttendee(rpcMessage, sender, channel, dbActorRef, validationError(s"User doesn't belong to the requested lao $laoId"))
-          _ <- checkIdentifierProof(
+          result <- checkIdentifierProof(
             rpcMessage,
             authenticate.identifier,
             authenticate.identifierProof,
             authenticate.nonce,
             validationError("Failed to verify the identifier proof with the given identifier/nonce pair")
           )
-          result <- checkNoDuplicateIdentifiers(
```

Comment:
Not really, it was removed here because the check was moved to `PopchaHandler` (in `registerAuthentication` to be precise) to avoid redundant checks and calls to the DB that would otherwise need to be duplicated in both the handler and the verifier.